mlflow/__init__.py

```diff
@@ -66,14 +66,16 @@
 
 _configure_mlflow_loggers(root_module_name=__name__)
 
-if sys.version_info.major == 2:
+_version_to_deprecate = (3, 5)
+if sys.version_info[:2] == _version_to_deprecate:
     warnings.warn(
-        "MLflow support for Python 2 is deprecated and will be dropped in a future "
-        "release. At that point, existing Python 2 workflows that use MLflow will "
-        "continue to work without modification, but Python 2 users will no longer "
-        "get access to the latest MLflow features and bugfixes. We recommend that "
-        "you upgrade to Python 3 - see https://docs.python.org/3/howto/pyporting.html "
-        "for a migration guide.",
+        "MLflow support for Python {ver} is deprecated and will be dropped in "
+        "MLlfow 1.15.0 release. At that point, existing Python {ver} workflows "
```

Comment:
```suggestion
        "an upcoming release. At that point, existing Python {ver} workflows "
```

I'm a fan of keeping the wording generic here in case we decide to change our mind for whatever reason :)

Reply:
Makes sense. I'll make the change